controller/src/main/java/io/pravega/controller/server/rest/resources/StreamMetadataResourceImpl.java

```diff
@@ -141,7 +142,40 @@ private void authenticate(String resourceName, AuthHandler.Permissions level) th
             String credentials = authParams.get(0);
             assert credentials != null;
 
-            if (!pravegaAuthManager.authenticate(resourceName, credentials, level)) {
+            if (!pravegaAuthManager.authenticateAuthorize(resourceName, credentials, level)) {
+                throw new AuthorizationException("Auth failed for " + resourceName, 403);
```

Comment:
Isn't there a class somewhere that defines these response codes somewhere? I don't like having hardcoded numbers throughout the codebase.

Reply:
Fixed. Using HTTP Status.